get_CLN_data.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+chain=$1
+color_green=$2
+color_red=$3
+ln_git_version=$4
+
+
+echo -ne '\r### Loading CoreLN data \r'
+
+ln_running=$(systemctl is-active cln)
+coreln_color="${color_green}"
+if [ -z "${ln_running##*inactive*}" ]; then
+  ln_running="down"
+  coreln_color="${color_red}\e[7m"
+else
+  if [ -z "${ln_running##*failed*}" ]; then
+    ln_running="down"
+    coreln_color="${color_red}\e[7m"
+  else
+    ln_running="up"
+  fi
+fi
+lncli="/home/cln/lightning/cli/lightning-cli"
+
+printf "%0.s#" {1..63}
+echo -ne '\r### Loading LND data \r'
+
+
+alias_color="${color_grey}"
+ln_alias="$(${lncli} getinfo | jq -r '.alias')" 2>/dev/null
+# jq '[.duration] | add'
+ln_walletbalance="$(${lncli} listfunds | jq -r '.outputs[0].value | tonumber')" 2>/dev/null
```

Comment:
If wallet is emtpy outputs[0] will be null. tonumber can't be used on null --> throws an error

Solution could be:
 ln_walletbalance=0
 if [ -z "$(${lncli} listfunds | jq -r '.outputs[0].value')" ];then
    ln_walletbalance=$(${lncli} listfunds | jq -r '.outputs[0].value | tonumber')
 fi

Reply:
Implemented just that, but replaced `-z` with `-n`
https://github.com/raspibolt/raspibolt-pulse/pull/2/commits/0e11684809c1cd68d5b613fbab4e5f640304a18c